app-info/eos-extra.xml

```diff
@@ -29,7 +29,7 @@
     <kudos>
       <kudo>GnomeSoftware::popular</kudo>
     </kudos>
-    <bundle type="flatpak"/>
+    <bundle type="flatpak">app/com.calibre_ebook.calibre/x86_64/stable</bundle>
```

Comment:
What about aarch64?

(IIRC this is why Umang left these out in the first place.)

Reply:
Mentioned in the commit message. I believe it won’t cause an issue, as gnome-software uses the data to
 1. call `gs_app_add_source()`, which adds an additional source (so won’t overwrite any aarch64 source)
 2. call `gs_app_set_branch()`, only using the branch part of the ref tuple

However, I have not tested this on aarch64, and that testing would need to be done.

Reply:
(I am trying to test it now)

Reply:
Yeah, it seems to work. I tested on an x86_64 VM by:
```
s/x86_64/aarch64/ in eos-extra.xml
scp eos-extra.xml tecnocode.co.uk:  # the external-appstream plugin will only accept https: URIs, not file: URIs
gsettings set org.gnome.software external-appstream-urls "['https://tecnocode.co.uk/eos-extra.xml']"
pkill gnome-software && sudo rm /var/cache/app-info/*/*
```
then opened gnome-software again and saw that the featured apps were still correctly listed on the main page. I tried installing one of them and that worked.

Reply:
Weird!